src/desktop/apps/artwork/components/facebook/index.jade

```diff
@@ -0,0 +1,18 @@
+script(type='text/javascript').
+  !function (f, b, e, v, n, t, s) {
+    if (f.fbq) return; n = f.fbq = function () {
+      n.callMethod ?
+        n.callMethod.apply(n, arguments) : n.queue.push(arguments)
+    };
+    if (!f._fbq) f._fbq = n; n.push = n; n.loaded = !0; n.version = '2.0';
+    n.queue = []; t = b.createElement(e); t.async = !0;
+    t.src = v; s = b.getElementsByTagName(e)[0];
+    s.parentNode.insertBefore(t, s)
+  }(window, document, 'script',
+    'https://connect.facebook.net/en_US/fbevents.js');
+  fbq('init', '1595366074113308');
```

Comment:
ENV var/magic num?

Reply:
pixel id - what's the convention for this? should it be a sharify thing? is `sd` available inside this jade template?

Reply:
eg i see similar stuff in sd like `sd.CRITEO_AUCTIONS_ACCOUNT_NUMBER`

Reply:
Yep `sd` is available in our Jade templates. There's also a process to make ENV vars (that are known to be safe client-side), available to sharify, via https://github.com/artsy/force/blob/24a5b31f3a5c20d56446b97583abb6c947831e14/src/lib/setup_sharify.js

I'm assuming this id is probably fine to be made available client-side.

So you can set it in the ENV, and propagate it to the [config](https://github.com/artsy/force/blob/24a5b31f3a5c20d56446b97583abb6c947831e14/src/desktop/config.coffee), and then include it in sharify, where it will be available inside the jade template.

Reply:
great - thanks for guiding me through it! the guideline is to include it in the header so i don't think we have much choice around exposing to clients.